types/artifacts.d.ts

```diff
@@ -436,65 +436,64 @@ declare global {
-        naturalWidth?: number;
+        /** The raw width attribute of the image element. CSS images will be set to null. */
+        attributeWidth: string | null;
+        /** The raw height attribute of the image element. CSS images will be set to null. */
+        attributeHeight: string | null;
         /**
-         * The natural height of the underlying image, uses img.naturalHeight. See https://codepen.io/patrickhulce/pen/PXvQbM for examples.
-         * TODO: explore revising the shape of this data. https://github.com/GoogleChrome/lighthouse/issues/12077
+         * The natural width and height of the underlying image resource, uses img.naturalHeight/img.naturalWidth. See https://codepen.io/patrickhulce/pen/PXvQbM for examples.
+         * Set to `undefined` if the data could not be collected.
          */
-        naturalHeight?: number;
-        /** The raw width attribute of the image element. CSS images will be set to the empty string. */
-        attributeWidth: string;
-        /** The raw height attribute of the image element. CSS images will be set to the empty string. */
-        attributeHeight: string;
-        /**
-         * The CSS width property of the image element.
-         * TODO: explore deprecating this in favor of _privateCssSizing. https://github.com/GoogleChrome/lighthouse/issues/12077
-         */
-        cssWidth?: string;
-        /**
-         * The CSS height property of the image element.
-         * TODO: explore deprecating this in favor of _privateCssSizing
-         */
-        cssHeight?: string;
+        naturalDimensions?: {
+          width: number;
+          height: number;
+        };
         /**
-         * The width/height of the element as defined by matching CSS rules. Set to `undefined` if the data was not collected.
-         * TODO: Finalize naming/shape of this data prior to Lighthouse 8. https://github.com/GoogleChrome/lighthouse/issues/12077
+         * The width/height of the element as defined by matching CSS rules.
+         * These are distinct from the `computedStyle` properties in that the are the raw property value.
+         * e.g. `width` would be `"100vw"`, not the computed width in pixels.
+         * Set to `undefined` if the data was not collected.
          */
-        _privateCssSizing?: {
+        cssEffectiveRules?: {
           /** The width of the image as expressed by CSS rules. Set to `null` if there was no width set in CSS. */
           width: string | null;
           /** The height of the image as expressed by CSS rules. Set to `null` if there was no height set in CSS. */
           height: string | null;
           /** The aspect ratio of the image as expressed by CSS rules. Set to `null` if there was no aspect ratio set in CSS. */
           aspectRatio: string | null;
-        }
+        };
+        /**
+         * The computed style
+         */
+        computedStyle: {
+          /** The CSS position attribute of the element */
```

Comment:
```suggestion
          /** `position` CSS property. */
```

Or remove (the name is self-explanatory).

Reply:
these are just moved, so they're also fine 🤷